tutorials/sinon/check-arguments.md

```diff
@@ -0,0 +1,95 @@
+To check what arguments a stub was called with, you have several options as stubs are treated as spies in sinon.js.
+
+
+## calledWith()
+
+`calledWith` will return true if the stub was called at least once with the provided arguments.
+The function can also return true if the stubbed function had more arguments than it should have, but are correctly positioned.
+
+```javascript
+const sinon = require('sinon');
+
+module.exports = {
```

Comment:
Why `module.exports`? Let's just define a separate object `const math = { add: (a, b) => { ... } }` instead of `module.exports` to avoid coupling this to Node.js